zerver/migrations/0498_stream_stream_topic_view_policy.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.2.11 on 2024-03-26 06:51
+
+from django.db import migrations, models
+
+STREAM_POST_POLICY_EVERYONE = 1
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("zerver", "0497_stream_is_support_stream"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="stream",
+            name="stream_topic_view_policy",
+            field=models.PositiveSmallIntegerField(default=STREAM_POST_POLICY_EVERYONE),
```

Comment:
This should probably be a `UserGroup` foreign key.

Reply:
I used `NamedUserGroup` instead. Check this out https://github.com/zulip/zulip/pull/29520#discussion_r1600628215.